src/Microsoft.AspNet.Mvc.TagHelpers/ValidationSummaryTagHelper.cs

```diff
@@ -8,10 +8,10 @@
 namespace Microsoft.AspNet.Mvc.TagHelpers
 {
     /// <summary>
-    /// <see cref="ITagHelper"/> implementation targeting &lt;div&gt; elements with an <c>asp-validation-summary</c>
+    /// <see cref="ITagHelper"/> implementation targeting any HTML element with an <c>asp-validation-summary</c>
     /// attribute.
     /// </summary>
-    [TargetElement("div")]
+    [TargetElement(Attributes = ValidationSummaryAttributeName)]
```

Comment:
this change in behaviour should be separate. discuss w/ @DamianEdwards as to whether we need to make this change.

note as well that there aren't that many block elements in HTML.  `<div>` was one of the few where splitting out the list made sense.